File: BAEKJOON/C/1_inout/Prob_2588.c
#include <stdio.h>

int main()
{

	int num1 = 0;
	int num2 = 0;
	int b = 0;
	int a = 0;
	int b1 = 0;
	int b10 = 0;
	int b100 = 0;

	scanf("%d %d", &num1, &num2);		//	scanf ¾µ‹š &ÇÊ¿ä

	b = num1;
	a = num2;

	b1 = b % 10;
	b10 = (int)((b % 100 - b1) / 10);
	b100 = (int)((b % 1000 - b % 100) / 100);

	printf("%d %d %d %d\n", b1 * a, b10 * a, b100 * a, b * a);
    
}

File: BAEKJOON/Python/1_inout/Prob_2588.py
a,b = map(int, input().split())

# a1 = a%10                 # 2
# a10 = (a%100 - a1)/10     # 7
# a100 = (a%1000 - a10)/100 # 4

b1 = b%10                 # 5
b10 = (int)((b%100 - b1)/10)     # 8
b100 = (int)((b%1000 - b%100)/100) # 3

print(b1*a)
print(b10*a)
print(b100*a)
print(b*a)
